src/index.html

```diff
@@ -9,6 +9,22 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="menu">
+      <a class="logo" href="#home">
+        <img src="images/logo.png" alt="">
+      </a>
+      <nav>
+        <ul class="nav__links"> 
+          <li><a href="">Apple</a></li>
```

Comment:
```suggestion
          <li><a href="#">Apple</a></li>
```
add classes so you can select them in css instead of tags